ovos_config/utils.py

```diff
@@ -3,8 +3,8 @@
 import time
 from os.path import dirname
 
-from watchdog.observers import Observer
```

Comment:
Trivial change could be reverted

Reply:
code autoformat, ordering imports for pep8